chia/_tests/blockchain/test_blockchain.py

```diff
@@ -3347,7 +3400,12 @@ async def test_long_reorg(
             assert pre_validation_results[i].error is None
             if (block.height % 100) == 0:
                 print(f"main chain: {block.height:4} weight: {block.weight}")
-            (result, err, _) = await b.add_block(block, pre_validation_results[i], None, sub_slot_iters=ssi)
+
+            fork_info: Optional[ForkInfo] = ForkInfo(block.height - 1, block.height - 1, block.prev_header_hash)
```

Comment:
thanks i originally did all this to avoid using fork_2 which needs to be optional and then opted for just doing new variable per case so its easier to read but i dont mind, changed